src/components/Budget/BudgetLineitem/__test__/BudgetLineItem.test.tsx

```diff
@@ -208,25 +208,30 @@ beforeEach(() => {
 	});
 });
 
-const inputIds = budgetLineItemInputFields;
+const budgetLineItemInputIds = budgetLineItemInputFields;
 
 describe("Budget Line Item Dialog tests", () => {
 	test("Mock response", async () => {
-		// 	for (let i = 0; i < inputIds.length; i++) {
-		// 		let fieldName = (await dialog.findByTestId(inputIds[i].id)) as HTMLInputElement;
-		// 		let value = intialFormValue[inputIds[i].key];
-		// 		await act(async () => {
-		// 			await fireEvent.change(fieldName, { target: { value } });
-		// 		});
-		// 		await expect(fieldName.value).toBe(value);
-		// 	}
-		// 	await act(async () => {
-		// 		let saveButton = await dialog.getByTestId("createSaveButton");
-		// 		expect(saveButton).toBeEnabled();
-		// 		fireEvent.click(saveButton);
-		// 		await wait();
-		// 	});
-		// 	await new Promise((resolve) => setTimeout(resolve, 1000));
-		// 	expect(creationOccured).toBe(true);
+		await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
What do we need to wait for 1s ? waiting for dom update / data loading ?

Reply:
I have removed this